packages/site/src/providers/AtlantisPreviewEditorProvider.tsx

```diff
@@ -360,23 +360,28 @@ const skeletonHTML = (theme: Theme, type: "web" | "mobile") => {
 <head>
 <style>
 html,body,#root {
-  height: 100%;
-  width: 99%;
+  height: 100vh;
+  width: 100vw;
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  min-height: 200px;
+  min-height: 280px;
+  margin: 0;
 }
   html[data-type="mobile"] {
-  border-radius:10px;
+  border-radius: var(--radius-base);
+}
+#root {
+  box-sizing: border-box;
+  padding: var(--space-smallest);
```

Comment:
This insets the wrapper of the component by _just_ a pinch to prevent the last/first pixels from being clipped by the parent iframe.

Doing this here instead of on the `body, html, #root` selector above because it had slightly unpredictable results, dependent on the components' own box-sizing/layout. For example, Card was fine, but DataTable was being clipped.

This approach is more specific to where we actually require this styling.

![image](https://github.com/user-attachments/assets/32e3d20c-8689-435f-aad7-ade67a5b63ab)
![image](https://github.com/user-attachments/assets/a289c2cb-b2fe-4e1f-8d6a-07b98d1e913d)
![image](https://github.com/user-attachments/assets/9a44cc12-8f68-4c21-92bb-9d9fb5c62a7d)